src/plugins/data/public/ui/query_editor/_query_editor.scss

```diff
@@ -137,7 +137,7 @@
 .osdQueryEditor__header {
   display: flex;
   align-items: center;
-  padding: 0 $euiSizeXS $euiSizeS;
+  padding: $euiSizeS $euiSizeXS 0;
```

Comment:
Will it impact the styles for other languages that may have customized header?

Reply:
I tested with this and I think it should be fine since I just move the padding from top to bottom